pystan/api.py

```diff
@@ -125,7 +125,11 @@ def stanc(file=None, charset='utf-8', model_code=None, model_name="anon_model",
 
     result = pystan._api.stanc(model_code_bytes, model_name_bytes)
     if result['status'] == -1:  # EXCEPTION_RC is -1
-        error_msg = "Failed to parse Stan model '{}'. Error message:\n{}".format(model_name, result['msg'])
+        msg = result['msg']
+        if PY2:
+            # fix problem with unicode in error message in PY2
+            msg = msg.encode('ascii', 'replace')
```

Comment:
This seems odd to me but it's been so long since I've programmed in Python 2.7 that I can't recall the compatibility tricks.